.pylint.ini

```diff
@@ -123,14 +123,23 @@ disable=
     no-else-return,
     logging-format-interpolation,
     duplicate-code,
+    unnecessary-lambda,
+    using-constant-test,
+    C0122,
+    C0302,
+    C0321,
+    R0201,
+    W0108
+    W0125,
+    W0201,
+    W0613
```

Comment:
Please use the name of the rule rather than the ID.

With CII, we're need to be deliberate here: https://bestpractices.coreinfrastructure.org/en/projects/754